tests/main.c

```diff
@@ -471,6 +471,40 @@ void test_stringRemove() {
     printGreen("test_stringRemove\n");
 }
 
+void test_stringCapitalize() {
+    TString simpleTString = stringInitWithCharArr("hello World! testing capitalize");
+    TString spaceThenDigitsTString = stringInitWithCharArr(" 123hello world! ");
+    TString specialSignThenDigitsTestString = stringInitWithCharArr("!123hello world! 0testing capitalize");
+
+    TString expectedSimpleTString = stringInitWithCharArr("Hello World! Testing Capitalize");
+    TString expectedSpaceThenDigitsTString = stringInitWithCharArr(" 123hello World! ");
+    TString expectedSpecialSignThenDigitsString = stringInitWithCharArr("!123hello World! 0testing Capitalize");
+
+    stringCapitalize(&simpleTString);
+    stringCapitalize(&spaceThenDigitsTString);
+    stringCapitalize(&specialSignThenDigitsTestString);
+
+    assertEq(stringLen(simpleTString), stringLen(expectedSimpleTString));
```

Comment:
Is it necessary to check the length? There is a [function](https://github.com/Tnirpps/cstring/blob/7ffbce5ba59e106cf3d0f972f64f3112dee03492/cstring.h#L274)